recipes/sail/all/patches/0.9.0-pre22-avif-find-package.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/src/sail-codecs/avif/avif.cmake b/src/sail-codecs/avif/avif.cmake
+index 6dfd3712..790fab62 100644
+--- a/src/sail-codecs/avif/avif.cmake
++++ b/src/sail-codecs/avif/avif.cmake
+@@ -1,13 +1,12 @@
+ macro(sail_find_dependencies)
+-    find_library(AVIF_LIBRARY avif)
+-    find_path(AVIF_INCLUDE_DIRS avif/avif.h)
++    find_package(libavif)
+ 
+-    if (NOT AVIF_LIBRARY OR NOT AVIF_INCLUDE_DIRS)
++    if (NOT libavif_FOUND)
+         return()
+     endif()
+ 
+-    set(sail_avif_include_dirs ${AVIF_INCLUDE_DIRS})
+-    set(sail_avif_libs ${AVIF_LIBRARY})
++    set(sail_avif_include_dirs ${avif_INCLUDE_DIRS})
++    set(sail_avif_libs ${avif_LIBRARY})
```

Comment:
And how do you link avif afterwards?

Reply:
With `target_link_libraries(target PRIVATE ${avif_LIBRARY})`. However, it looks like `${avif_LIBRARY}` is not the correct avif library name for all types of builds. It works only on static builds apparently pointing to the static `avif.lib`. Can you recommend the correct name to link against?

Reply:
Thank you so much, @SpaceIm ! it would be impossible to finish the recipe without your help.